pc-bufsw.el

```diff
@@ -210,6 +210,13 @@ Formatting can be added using text properties, e.g.:
     :type '(choice (const nil)
                    face))
 
+  (defcustom pc-bufsw-prefer-current-window nil
+    "Prefer buffers previously displayed in the current window when building buffer list."
+    :type 'boolean)
+  (defcustom pc-bufsw-prefer-current-frame nil
```

Comment:
Make this an enum with two values, not a boolean. I would like to add a third option there later not to show buffers from other frames at all.